asciidoctor-revealjs.gemspec

```diff
@@ -29,4 +29,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'thread_safe', '~> 0.3.5'
 
   s.add_development_dependency 'rake', '~> 10.4.2'
+  s.add_development_dependency 'asciidoctor-doctest', '~> 2.0.0.beta.4'
```

Comment:
I'm a cybersecurity person and I think it's always better to enable upstream projects to patch their issues but I see your point. I'll pin to that version but I hope you'll keep us in the loop for improvements / security issues :wink: